### DISCUSSION TITLE
[A-L] (14 Luglio 2023) Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#396] [Esami A-L]
Mario, Claudia e Gianni hanno fatto un test di ammissione. Rilasciano le seguenti dichiarazioni.

- Mario dice: "O io ho superato il test e Gianni non lo ha superato o Gianni e Claudia hanno superato il test." 

- Claudia dice: "O Mario e Gianni superano entrambi il test o nessuno dei due lo ha superato."

- Gianni dice: "Se Mario non ha superato il test allora Claudia ha superato il test."

---

1. Formalizzare le tre affermazioni in logica proposizionale scegliendo un linguaggio adeguato.

2. E possibile che tutti e tre dicano il vero? Argomentare.

3. Se tutti dicono il falso, posso dire chi ha superato sicuramente il test? Argomentare.

(Indicazione: usare le tavole di verità).

**Comment by cek753** [comment]
<img width="491" height="440" alt="image" src="https://github.com/user-attachments/assets/3b2ca3a6-c092-43b2-9a8f-fb2ad9660bcd" />
